addons/mass_mailing_sms/controllers/main.py

```diff
@@ -43,9 +43,11 @@ def blacklist_number(self, mailing_id, trace_code, **post):
             return request.redirect('/web')
         # parse and validate number
         sms_number = post.get('sms_number', '').strip(' ')
+        country_phone_code = request.env['res.country'].search([('code', '=', request.geoip.country_code)]).phone_code
```

Comment:
Could say
```suggestion
        country = request.env['res.country'].search([('code', '=', request.geoip.country_code)])
```
Then use country.code and country.phone_code when calling phone_format